resources/carrier/en/385.txt

```diff
@@ -12,11 +12,35 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
+# 385 - Croatia
+
+# Source(s):
+# -----------------------
+# HAKOM: https://www.hakom.hr/default.aspx?id=817
+# -----------------------
+
+# Carriers:
+# -----------------------
+# Hrvatski Telekom: hrvatskitelekom.hr - Hrvatski Telekom d.d.
+# Vip: vipnet.hr - VIPnet d.o.o.
+# Tele2: tele2.hr - Tele2 d.o.o.
+# bonbon: bonbon.hr - Hrvatski Telekom d.d.
+# Telefocus: telefocus.hr - TELEFOCUS d.o.o.
+# -----------------------
+
+# -----------------------
+# Last checked: 6/12/2017
+# Last updated: 6/12/2017
+# -----------------------
+
```

Comment:
We do nota maintain all this info here. Could you please replace all these new lines with below line.
```
# Source: https://www.hakom.hr/default.aspx?id=817
# bonbon is a virtual network operator.
```
The actual carrier mapping updates looks good to me.